x/avs/keeper/keeper.go

```diff
@@ -44,47 +46,83 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 }
 
 func (k Keeper) AVSInfoUpdate(ctx sdk.Context, params *AVSRegisterOrDeregisterParams) error {
+	avsInfo, _ := k.GetAVSInfo(ctx, params.AvsAddress)
+
+	action := params.Action
+
+	if action == RegisterAction && avsInfo != nil {
+		return errorsmod.Wrap(types.ErrAlreadyRegistered, fmt.Sprintf("the error input arg is:%s", params.AvsAddress))
```

Comment:
can replace this with specific error  " the avsaddress is :%s",

Reply:
fixed